ko/api/jdbc.rst

```diff
@@ -667,15 +667,15 @@ OID를 사용할 때 다음의 규칙을 지켜야 한다.
                CUBRIDResultSet rs = (CUBRIDResultSet)stmt.executeQuery(sql);
                while (rs.next ()) {
                    CUBRIDOID oid = rs.getOID(1);
-                   oid.addToSet("settest",new Integer(10));
-                   oid.addToSet("multisettest",new Integer(20));
-                   oid.addToSequence("listtest",1,new Integer(30));
-                   oid.addToSequence("listtest",100,new Integer(100));
-                   oid.putIntoSequence("listtest",99,new Integer(99));
-                   oid.removeFromSet("settest",new Integer(1));
-                   oid.removeFromSet("multisettest",new Integer(2));
-                   oid.removeFromSequence("listtest",99);
-                   oid.removeFromSequence("listtest",1);
+				   oid.addToSet("settest", Integer.valueOf(10));
+				   oid.addToSet("multisettest", Integer.valueOf(20));
+				   oid.addToSequence("listtest", 1, Integer.valueOf(30));
+				   oid.addToSequence("listtest", 100, Integer.valueOf(100));
+				   oid.putIntoSequence("listtest", 99, Integer.valueOf(99));
+				   oid.removeFromSet("settest", Integer.valueOf(1));
+				   oid.removeFromSet("multisettest", Integer.valueOf(2));
+				   oid.removeFromSequence("listtest", 99);
+				   oid.removeFromSequence("listtest", 1);
```

Comment:
please fix the broken indentation in this example. you need to build the manual and check on a web browser after you revise.

Reply:
I check it and commit after build the manual.